### PR TITLE
feat: BREAKING CHANGES: update to pubsub 2.5.0 (GA) (CUR-928)

### PR DESCRIPTION
**Pull request by mgabeler-lee-6rs**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -53,13 +53,13 @@ jobs:
...
     docker:
-      - image: circleci/node:10.15.3
+      - image: circleci/node:12.16.3
         environment:
           MOCHA_OPTS: --reporter mocha-junit-reporter --reporter-options mochaFile=./reports/junit/mocha/mocha.xml
       - image: singularities/pubsub-emulator
         environment:
           PUBSUB_PROJECT_ID: pubcap-test
-          PUBSUB_LISTEN_ADDRESS: 0.0.0.0:8085
+          PUBSUB_LISTEN_ADDRESS: 0.0.0.0:8802
```
Comment (AshleyFoster) [inline .circleci/config.yml:62]: Just curious what was the reason for changing the listen address?

Comment (mgabeler-lee-6rs) [reply .circleci/config.yml:62]: So that it matches what we use in `6mon`, so you can run `npm test` on localhost :)

Comment (AshleyFoster) [reply .circleci/config.yml:62]: Oh, rad! That will be super useful. 💪

**Review by AshleyFoster — APPROVED** [review]
Looks good to me!

**Pull request merged**